client/src/commonMain/kotlin/com/algolia/search/configuration/ConfigurationSearch.kt

```diff
@@ -3,6 +3,7 @@
 package com.algolia.search.configuration
 
 import com.algolia.search.client.ClientSearch
+import com.algolia.search.configuration.internal.*
```

Comment:
```suggestion
import com.algolia.search.configuration.internal.*
import com.algolia.search.configuration.internal.DEFAULT_CONNECT_TIMEOUT
```

I think you missed this one?

Reply:
ah actually the star allows it already